namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-staging/resources/redis.tf

```diff
@@ -0,0 +1,46 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "redis_elasticache" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=4.0"
```

Comment:
Unless there is a reason not to, could you please use the latest version of the module ?  ([here](https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/releases/tag/4.1))

Reply:
I've updated it to use 4.1